Bot can recive write requests from system app.js

Achoobert opened the pull request:
Will need to have tests written before is complete

Achoobert committed:
[add] setup file structure. IS STILL VERY BROKEN

Due to "Host update server not active." cannot get the command through to the processcheck command

Achoobert committed:
[fixed] is in progress boolians

Achoobert committed:
[add] tracks all PIDs on Watch and Warn lists and Writes messages to slack about them

LACKS variables for determining which processes need to be tracked
Has: removing PIDs from lists after time and because CPU is at a particular level

Achoobert committed:
[fix] Removed unnecessary files as command is no longer relivant

Hiro-Nakamura commented on config/bot_manager.js:
```diff
@@ -66,9 +66,9 @@ module.exports = {
-    //   port: "1338",
-    //   accessToken: "li77lePigLi77lePigLe7MeIn"
-    // }
+    tcp: {
```
since this is in source control, don't commit this, since we don't want to accidentally install this in a running instance.

Achoobert committed:
[fix] Requested Changes, perform check and source control

performing the if (.enable) { check before the if (data.indexOf()) check
Commented out "source control" lines
